src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-kubernetes-proxy.mdx

```diff
@@ -9,87 +9,22 @@ redirects:
   - /docs/integrations/kubernetes-integration/installation/configure-kubernetes-proxy
 ---
 
-If you are running our Kubernetes integration with a proxy, you need to configure each component (the [infrastructure agent](/docs/infrastructure/install-infrastructure-agent), logging, Kubernetes events, etc.) to correctly work with your proxy.
+If you are running our Kubernetes integration with a proxy, you need to configure each component (the [infrastructure agent](/docs/infrastructure/install-infrastructure-agent), logging and synthetics minion) to correctly work with your proxy.
```

Comment:
Should we add the NR metric adapter to the list of components that needs to be configured?

Reply:
It is configured with the `global.proxy` in the same way as other integrations: https://github.com/newrelic/docs-website/pull/7322/files/c60847485e721d597c4ac64da1305f453ee31198#diff-aeb31ebad1a4a4bbe9ea7f906d3365355802c18e20b2a7be741a56f76cc328ffR22